demo/src/test/java/com/example/demo/repository/EngineerRepositoryTest.java

```diff
@@ -0,0 +1,142 @@
+package com.example.demo.repository;
+
+import com.example.demo.bean.Engineer;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+class EngineerRepositoryTest {
+
+    private static final EngineerRepository engineerRepository = new EngineerRepository();
+    private Engineer engineer1;
+    private Engineer engineer2;
+    private Engineer engineer3;
+    private Engineer engineer4;
+    private Engineer engineer5;
+    private Engineer engineer1i;
+    private Engineer engineer2i;
+    private Engineer engineer3i;
+    private Engineer engineer4i;
+    private Engineer engineer5i;
+    private Engineer engineer5o = new Engineer("5", "Нестор", "Майко", "Инжинер по гарантии", "высшая", "высшее", 5, 10);
+    private Engineer engineer4o = new Engineer("4", "Денис", "Пелля", "Мастер цеха", "высшая", "высшее", 3, 7);
+    private Engineer engineer3o = new Engineer("3", "Юрий", "Яковлев", "Главный инжинер", "высшая", "высшее", 1, 10);
+    private Engineer engineer2o = new Engineer("2", "Роман", "Валюшко", "Инжинер по гарантии", "высшая", "высшее", 5, 10);
+    private Engineer engineer1o = new Engineer("1", "Роман", "Валюшко", "Главный менеджер", "высшая", "высшее", 5, 10);
+    private final Engineer engineer5d = new Engineer("4", "Роман", "Валюшко", "Главный менеджер", "высшая", "высшее", 5, 10);
+
+    @BeforeEach
+    public void before() {
```

Comment:
Maybe add a method:
public void setup()

Reply:
Why is "public void setup()" better than "public void before()"?